hardware/tests/test_benchmark_mem.py

```diff
@@ -122,17 +111,6 @@ def test_check_mem_size(self, mock_popen, mock_cpu_socket,
         for block_size in block_size_list:
             self.assertFalse(mem.check_mem_size(block_size, 2))
 
-    def test_run_sysbench_memory_forked_text(self, mock_popen, mock_cpu_socket,
```

Comment:
ditto

Reply:
because the output of commands in Python 3.x is bytes, not pure text, so we don't need this, we already have the correspondent bytes test